### ISSUE TITLE
Could not load structure type "overview"

### ISSUE THREAD

**Issue by BorislavSabev** [sulu/sulu-minimal#39]
While following the [getting started/installation guide](http://docs.sulu.io/en/latest/book/getting-started.html) I got the following error when running `bin/adminconsole sulu:build dev`:

```
[Sulu\Component\Content\Metadata\Factory\Exception\StructureTypeNotFoundException]
Could not load structure type "overview" for document type "home", looked in "__PATHOBDUSCATED__/SuluTest2/app/Resources/templates/pages"  
[InvalidArgumentException]                                                                                                          
The file "overview.xml" does not exist (in: __PATHOBDUSCATED__/SuluTest2/app/Resources/templates/pages)
```

The dev build did not finish and my test project was broken. I was able to solve this by copying `app/Resources/pages/overview.xml.dist`  **from sulu/sulu-standard** to `app/Resources/templates/pages/overview.xml` of my `sulu/sulu-minimal` install. 

I am not sure if the file I copied has exactly the same contents, but this solved my problem.
Is there a problem in `sulu/sulu-minimal` or did I not copy the right version from composer?
I am an experienced developer and I followed the guide line by line. I did not modify your composer commnads.

**Comment by BorislavSabev** [comment]
To add some info: in the admin app when I try to create a page I get a Twig missing template error.
```
Unable to find template "ClientWebsiteBundle:templates:overview.html.twig" (looked into: SuluTest2/app/Resources/views, SuluTest2/vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form).
```

**Comment by danrot** [comment]
I just tested it, and it works for me... In sulu-minimal there are only two templates, none of them called overview, so it should not try to create a page with that template: https://github.com/sulu/sulu-minimal/tree/master/app/Resources/templates/pages

I have just tried it, and after executing `composer create-project sulu/sulu-minimal my-project -n` and `bin/console sulu:build dev` (both commands were successful) Sulu could be run with `bin/console server:run` without any errors.

Are you sure you don't have any leftovers from another installation, where an `overview.xml` was existing? Have you tried clear the Symfony cache and running `bin/console sulu:build dev --destroy` (the destroy option regenerates basically everything)?

**Comment by BorislavSabev** [comment]
@danrot I will retest it but I am absolutely sure I started in a clean directory tree. 

It is possible that Symphony/Composer cache was the cause as I did try installing `sulu/sulu-standard` in a totally different root path. Before. But why are the Symphony caches mixed?

**Comment by danrot** [comment]
@BorislavSabev They shouldn't be mixed... but if you have used the same database it might be possible that a reference to the `overview` template was stored there.

**Comment by BorislavSabev** [comment]
@danrot the DB was the same but all tables were dropped between attempts.
Let me try again.

**Comment by BorislavSabev** [comment]
@danrot OK I retested and it is fine. 
I suspect I did no clear the DB since I did several tests of sulu-minimal & sulu-standard and it is possible that I missed something in between.
 Thanks.
